java/client/src/main/java/glide/api/commands/SetBaseCommands.java

```diff
@@ -76,6 +76,22 @@ public interface SetBaseCommands {
      */
     CompletableFuture<Long> scard(String key);
 
+    /**
+     * Checks whether each member is contained in the members of the set stored at <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/smismember/">redis.io</a> for details.
+     * @param key The key of the set to check.
+     * @param members A list of members to check for existence in the set.
+     * @return An <code>array</code> of <code>boolean</code> values, each indicating if the respective
```

Comment:
Should this be Boolean?
```suggestion
     * @return An <code>array</code> of <code>Boolean</code> values, each indicating if the respective
```